go.mod

```diff
@@ -11,8 +11,8 @@ require (
 	github.com/flynn/noise v1.1.0
 	github.com/google/gopacket v1.1.19
 	github.com/gorilla/websocket v1.5.0
-	github.com/hashicorp/golang-lru/arc/v2 v2.0.5
-	github.com/hashicorp/golang-lru/v2 v2.0.5
+	github.com/hashicorp/golang-lru/arc/v2 v2.0.7
+	github.com/hashicorp/golang-lru/v2 v2.0.7
```

Comment:
Many of these changes to go.mod seem unrelated, and should not be folded into this PR.

Reply:
I reverted dependency upgrades that were not directly related to `quic-go`